packages/server/src/tests/epics/epicSelection.ts

```diff
@@ -190,6 +190,29 @@ export interface Result {
     debug?: Debug;
 }
 
+export const AppliedLearningBanditTestsNames = {
+    BanditTestEpsilon0: '2024-07-25_BANDIT_ABTEST_VARIANTS', // This is the AB test
```

Comment:
since we are checking with `includes` it will work if we leave out the date as well, then when they're defined in the RRCP the dates can be added there